src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -123,14 +123,14 @@ public void OperationParametersValidation()
             var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "operations-invalid-parameters.json"));
             messages.AssertOnlyValidationMessage(typeof(OperationParametersValidation));
         }
-        
```

Comment:
As @olydis suggested try editing in Visual studio so that we don't see this unnecessary diffs